packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/CameraSelectorHostApiImpl.java

```diff
@@ -52,12 +52,9 @@ public List<Long> filter(@NonNull Long identifier, @NonNull List<Long> cameraInf
     }
 
     List<CameraInfo> filteredCameraInfos = cameraSelector.filter(cameraInfosForFilter);
-    final CameraInfoFlutterApiImpl cameraInfoFlutterApiImpl =
-        new CameraInfoFlutterApiImpl(binaryMessenger, instanceManager);
     List<Long> filteredCameraInfosIds = new ArrayList<Long>();
 
     for (CameraInfo cameraInfo : filteredCameraInfos) {
-      cameraInfoFlutterApiImpl.create(cameraInfo, result -> {});
```

Comment:
Why did you remove the creation of the `CameraInfo`s? I'm not positive this is safe to do.

Reply:
This was because we are passing in CameraInfos that have already been added to the instance manager, and can re-use the ones that remain after filtering. Closing after offline discussion